### PR TITLE
Added Node.js quiz questions and answers.

### LINKED ISSUE #96: Add node js questions and answers.
Add node js questions and answers in the database.
Add link of the quiz in home page.

### PR DESCRIPTION
**Pull request by A-Meri**

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -43,6 +43,15 @@ const Header = () => {
               />
               JavaScript
             </NavDropdown.Item>
+            <NavDropdown.Divider />
+            <NavDropdown.Item as={Link} to="/node">
+              <FontAwesomeIcon
+                className="mr-2 text-warning"
+                aria-hidden="true"
+                icon={faJs}
```
Comment (bonboh) [inline client/src/components/Header.js]: Can we use a Node icon?

**Review by bonboh — CHANGES_REQUESTED** [review]
Just one question

File: client/src/components/Header.js (hunk 1 above)
Comment (A-Meri) [reply client/src/components/Header.js]: I found this link -  https://fontawesome.com/icons/node?style=brands
Would that be ok ?

Comment (bonboh) [reply client/src/components/Header.js]: Yes it's a node icon so it fits our needs

**Review by bonboh — APPROVED** [review]

**Pull request merged**